test/common/fixtures/plugins/coverage/kibana.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": "coverage-fixtures",
+  "id": "coverageFixtures",
```

Comment:
After merging master on Monday I faced this error in functional tests:

```
11:37:23       │ proc [kibana]  FATAL  Error: Failed to initialize plugins:
11:37:23       │ proc [kibana] 	Plugin "id" must be camelCase, but found: coverage-fixtures. (invalid-manifest, /dev/shm/workspace/kibana12/test/common/fixtures/plugins/coverage/kibana.json)
```

I changed plugin id according error an looks like it is working fine